bin/deploy.sh

```diff
@@ -4,26 +4,36 @@ PROJECT=super-big-data
 IMAGE=gcr.io/${PROJECT}/product-eng-webhooks
 
 env_vars="ENV=production,"
-env_vars="${env_vars}GH_APP_PRIVATE_KEY_FOR_GETSENTRY=${GH_APP_PRIVATE_KEY_FOR_GETSENTRY},"
-env_vars="${env_vars}GH_APP_PRIVATE_KEY_FOR_CODECOV=${GH_APP_PRIVATE_KEY_FOR_CODECOV},"
-env_vars="${env_vars}DD_API_KEY=${DD_API_KEY},"
-env_vars="${env_vars}DD_APP_KEY=${DD_APP_KEY},"
-env_vars="${env_vars}GH_WEBHOOK_SECRET=${GH_WEBHOOK_SECRET},"
-env_vars="${env_vars}SLACK_SIGNING_SECRET=${SLACK_SIGNING_SECRET},"
-env_vars="${env_vars}SLACK_BOT_USER_ACCESS_TOKEN=${SLACK_BOT_USER_ACCESS_TOKEN},"
 env_vars="${env_vars}VERSION=${VERSION},"
-env_vars="${env_vars}SENTRY_WEBPACK_WEBHOOK_SECRET=${SENTRY_WEBPACK_WEBHOOK_SECRET},"
 env_vars="${env_vars}DB_USER=${DB_USER},"
 env_vars="${env_vars}DB_NAME=${DB_NAME},"
-env_vars="${env_vars}DB_PASSWORD=${DB_PASSWORD},"
 env_vars="${env_vars}DB_INSTANCE_CONNECTION_NAME=${DB_INSTANCE_CONNECTION_NAME},"
-env_vars="${env_vars}GH_USER_TOKEN=${GH_USER_TOKEN},"
 
+secret_names="
+GH_APP_PRIVATE_KEY_FOR_GETSENTRY
+GH_APP_PRIVATE_KEY_FOR_CODECOV
+DD_API_KEY
+DD_APP_KEY
+GH_WEBHOOK_SECRET
+SLACK_SIGNING_SECRET
+SLACK_BOT_USER_ACCESS_TOKEN
+SENTRY_WEBPACK_WEBHOOK_SECRET
+DB_PASSWORD
+GH_USER_TOKEN
+"
+
+secrets=""
+for secret_name in $secret_names; do
```

Comment:
I don't think this would work, maybe something like would
```
echo "$secret_names" | while IFS= read -r secret_name; do
  secrets="${secrets}${secret_name}=eng-pipes-${secret_name},"
done
```

Reply:
Just checked on `ubuntu:latest` docker image with `/bin/sh` shell:
```
# cat 1.sh
#!/bin/sh

secret_names="
GH_APP_PRIVATE_KEY_FOR_GETSENTRY
GH_APP_PRIVATE_KEY_FOR_CODECOV
DD_API_KEY
DD_APP_KEY
GH_WEBHOOK_SECRET
SLACK_SIGNING_SECRET
SLACK_BOT_USER_ACCESS_TOKEN
SENTRY_WEBPACK_WEBHOOK_SECRET
DB_PASSWORD
GH_USER_TOKEN
"

secrets=""
for secret_name in $secret_names; do
	secrets="${secrets}${secret_name}=eng-pipes-${secret_name},"
done

echo "${secrets}"

# ./1.sh
GH_APP_PRIVATE_KEY_FOR_GETSENTRY=eng-pipes-GH_APP_PRIVATE_KEY_FOR_GETSENTRY,GH_APP_PRIVATE_KEY_FOR_CODECOV=eng-pipes-GH_APP_PRIVATE_KEY_FOR_CODECOV,DD_API_KEY=eng-pipes-DD_API_KEY,DD_APP_KEY=eng-pipes-DD_APP_KEY,GH_WEBHOOK_SECRET=eng-pipes-GH_WEBHOOK_SECRET,SLACK_SIGNING_SECRET=eng-pipes-SLACK_SIGNING_SECRET,SLACK_BOT_USER_ACCESS_TOKEN=eng-pipes-SLACK_BOT_USER_ACCESS_TOKEN,SENTRY_WEBPACK_WEBHOOK_SECRET=eng-pipes-SENTRY_WEBPACK_WEBHOOK_SECRET,DB_PASSWORD=eng-pipes-DB_PASSWORD,GH_USER_TOKEN=eng-pipes-GH_USER_TOKEN,
```

Reply:
got it, thanks for checking